Automattic/wp-calypso
# Logged-in Performance Profiler: Allow re-running test

Closes https://github.com/Automattic/dotcom-forge/issues/9138.

## Proposed Changes

Adds the ability to re-run a test from the dashboard. 

![image](https://github.com/user-attachments/assets/ab476a18-72e1-4271-9b20-64489e79444e)

## Testing Instructions

1. add this test report url to your WoA site 
`https://www.atliq.com/&hash=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6NjkzNH0.uoPBbETEfzOEqTdJ_kro-zUpmvSF6vQz7JabJYumwME`

`wp option update wpcom_performance_report_url "url"`

2. Go to `/site-performance/WoA_Site`
3. You should see a report returned based on the url that was added
4. You can now rerun the test using the `Test again` button
